Modules/Devices/src/Devices.Application/PushNotifications/Commands/DeleteDeviceRegistration/Handler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.PushNotifications;
+using Backbone.Modules.Devices.Application.PushNotifications.Commands.UpdateDeviceRegistration;
+using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
+using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications.Handles;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using MediatR;
+using ApplicationException = Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace Backbone.Modules.Devices.Application.PushNotifications.Commands.DeleteDeviceRegistration;
+internal class Handler : IRequestHandler<DeleteDeviceRegistrationCommand, Unit>
+{
+    private readonly IdentityAddress _activeIdentity;
+    private readonly IPushService _pushService;
+    private readonly DeviceId _activeDevice;
+
+    public Handler(IPushService pushService, IUserContext userContext)
+    {
+        _pushService = pushService;
+        _activeIdentity = userContext.GetAddress();
+        _activeDevice = userContext.GetDeviceId();
+    }
+
+
+    public async Task<Unit> Handle(DeleteDeviceRegistrationCommand request, CancellationToken cancellationToken)
+    {
+        Console.WriteLine("Handle DeleteDeviceRegistrationCommand");
```

Comment:
Do NOT use `Console.WriteLine`. If you want to log something, use the `ILogger`.

But something like this is logged automatically by the MediatR pipeline anyway (Enmeshed.BuildingBlocks.Application.MediatR.LoggingBehavior).